systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainST.java

```diff
@@ -69,6 +73,182 @@ void testProducerConsumer() {
         ClientUtils.waitForClientSuccess(OAUTH_CONSUMER_NAME, NAMESPACE, MESSAGE_COUNT);
     }
 
+    @Test
+    void testProducerConsumerAudienceTokenChecks() {
+        LOGGER.info("Update Kafka broker configuration to use OAuth2 access token audience checks");
```

Comment:
Wouldn't better to add this listener to Kafka created in beforeAll? It will save some time because we won't need replace Kafka for each test

Reply:
Updated, added listeners to default Kafka. All tests locally passed.